server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/eq/utils/HazardUtil.java

```diff
@@ -336,11 +336,13 @@ public static double getCorrectUnitsOfPGA(double pga, String units0, String unit
     public static double getCorrectUnitsOfPGD(double pgd, String units0, String units1) {
         if (units0 != null && units0.equalsIgnoreCase(units1)) {
             return pgd;
-        } else if (units_m.equalsIgnoreCase(units0) || "m".equalsIgnoreCase(units0) && units_ft.equalsIgnoreCase(units1)) {
+        } else if (units_m.equalsIgnoreCase(units0) || "m".equalsIgnoreCase(units0) && (units_ft.equalsIgnoreCase(units1) || "ft".equalsIgnoreCase(units1))) {
```

Comment:
Is this because we have inconsistent definition of feet vs ft?

Reply:
Yeah, there is some inconsistency in the static variable definitions (some are abbreviations and some aren't). Also, I wanted to match that a user could request "m" or "meters" so I made it so a user could request "feet" or with the abbreviation "ft".